### DISCUSSION TITLE
When using zfs-autobackup for strictly snapshots

### DISCUSSION THREAD

**Discussion by hput** [psy0rz/zfs_autobackup#131] [Q&A]
If I use zfs-autoback for snapshotting:  After having recursively marked all filessytems like"

    "zfs set autobackup:offsite1=true rpool {from getting started}

Only I used: zfs set absn:sn=true rpool p0 p1 
on the ssh-server host 

And then set this on the backup server: (host name oi) 
        /bin/zfs-autobackup -v --ssh-source 2x2 sn

What I want to know is ,over time, does zfs-autobackup figure out the snap names or do I have to do something in that regard.
The above will run once a day for a while until I see what it does.

**Comment by hput** [comment]
I made some very simpleminded errors in that question, The biggest and worse being blythly thinking I could change the autobackup part the tag with no problems... 
So unless someone has something to add, maybe just ignore this little brute

**Comment by psy0rz** [comment]
zfs-autobackup only recognizes the snapshots that correspond the the backup name. It will ignore the others.
